private-templates-service/client/src/components/NavBar/SearchBar/styled.tsx

```diff
@@ -1,8 +1,9 @@
 import styled from "styled-components";
 import { SearchBox } from "office-ui-fabric-react";
 import { BREAK, COLORS } from "../../../globalStyles";
+import * as STRINGS from "../../../assets/strings";
 
-export const StyledSearchBox = styled(SearchBox)`
+export const StyledSearchBox = styled(SearchBox).attrs({"ariaLabel": STRINGS.SEARCHBAR_DESCRIPTION})`
```

Comment:
Kody asked to move the ariaLabel from **syled.tsx** to its component in the actual code. **<StyledSearchBox ariaLabel="..">**

Reply:
Fixed!